app/server/lib/OIDCConfig.ts

```diff
@@ -29,6 +29,9 @@
  *    env GRIST_OIDC_IDP_SKIP_END_SESSION_ENDPOINT
  *        If set to "true", on logout, there won't be any attempt to call the IdP's end_session_endpoint
  *        (the user will remain logged in in the IdP).
+ *    env GRIST_OIDC_SP_FORCE_EMAIL_VERIFIED
```

Comment:
I prefer options that default to false. Wdyt of naming it GRIST_OIDC_SP_IGNORE_EMAIL_VERIFIED, with the explanation that by default, logins will only be accepted from users with `email_verified` claim from the identity provider set to true. Setting this flag will turn off this check.

Reply:
Good idea, done!